configurations/scylla_cloud_nemesis.yaml

```diff
@@ -1,4 +1,4 @@
-nemesis_class_name: ChaosMonkey
```

Comment:
this change is temporary, just for testing
I'm thinking if we should also take this file out of SCT, even that it would make it a bit harder to configure (since it wouldn't be in SCT path and using) 
@bentsi WDT, about having also test/config yamls move out ?

Reply:
better to move them out